Assets/Scripts/SS3D/Systems/Tile/TileResourceLoader.cs

```diff
@@ -9,13 +10,13 @@ namespace SS3D.Systems.Tile
     /// <summary>
     /// Loads assets used by the tilemap. Can be used to retrieve scriptableobjects from a name string.
     /// </summary>
-    public class TileResourceLoader: MonoBehaviour
+    public sealed class TileResourceLoader: MonoBehaviour
     {
         public Sprite _missingIcon;
 
-        public bool IsInitialized { get; private set; } = false;
+        public bool IsInitialized { get; private set; }
 
-        private List<GenericObjectSo> _assets;
+        public List<GenericObjectSo> Assets { get; private set; }
```

Comment:
why setting that public ?

Reply:
I removed the method GetAllAssets for the simple acessor to the Assets list